js/tji-datasets-explore.js

```diff
@@ -319,7 +337,8 @@ TJIChartView.prototype.create_filter_panel = function() {
   // Get a sorted list of all the unique values for each column
   _.each(this.filter_configs, function(filter_config) {
     var values = _.filter(_.sortBy(_.uniq(_.map(that.state.data, filter_config.name))), _.identity);
-    if (values.indexOf(that.missing_data_label) !== -1) {
+    // Move the special "missing data" label to the last position
+    if (~values.indexOf(that.missing_data_label)) {
```

Comment:
@wetchler 
![loveyou](https://user-images.githubusercontent.com/3039125/42714344-73f2cf28-86b8-11e8-84f0-5ec732734522.gif)

Reply:
If you insist on doing this (◔_◔), please also change the other places in this file that are using the `!== -1` style. I see two others.

Reply:
👍